### ISSUE TITLE
[2023.10.12-Beta] climate and water_heater could not be set up

### ISSUE THREAD

**Issue by ChrisMisker** [BenPru/luxtronik#149]
Hi,
I get this error in the notification:
<img width="477" alt="Scherm­afbeelding 2023-10-12 om 07 02 09" src="https://github.com/BenPru/luxtronik/assets/37826263/d9e04bee-e040-4c8a-9f4c-e48859481781">

When inspecting the logs, I get these errors:
```
Logger: homeassistant.setup
Source: setup.py:359
First occurred: 06:58:16 (2 occurrences)
Last logged: 06:58:16

Unable to prepare setup for platform luxtronik2.climate: Platform not found (Exception importing custom_components.luxtronik2.climate).
Unable to prepare setup for platform luxtronik2.water_heater: Platform not found (Exception importing custom_components.luxtronik2.water_heater).
```

```
Logger: homeassistant.loader
Source: loader.py:838
First occurred: 06:58:16 (1 occurrences)
Last logged: 06:58:16

Unexpected exception importing platform custom_components.luxtronik2.water_heater
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 838, in get_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 855, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/luxtronik2/water_heater.py", line 52, in <module>
    LuxtronikWaterHeaterDescription(
TypeError: LuxtronikWaterHeaterDescription.__init__() got an unexpected keyword argument 'temperature_unit'
```

```
Logger: homeassistant.loader
Source: loader.py:838
First occurred: 06:58:16 (1 occurrences)
Last logged: 06:58:16

Unexpected exception importing platform custom_components.luxtronik2.climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 838, in get_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 855, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/luxtronik2/climate.py", line 88, in <module>
    LuxtronikClimateDescription(
TypeError: LuxtronikClimateDescription.__init__() got an unexpected keyword argument 'temperature_unit'
```

I'm on 2023.10.12-Beta and on Home Assistant 2023.10.1. I've also restarted the Heatpump, just to be sure, didn't fix it. (And of course rebooted HA). I have also removed and reinstalled the custom component, didn't work.

**Comment by rhammen** [comment]
Sorry, my mistake, forgot to include 2 small changes in model.py

**Comment by ChrisMisker** [comment]
Seems to be fixed in latest beta (2023.10.13).
